language-server/modules/langserver-core/src/test/resources/codeaction/ignore-unused-var/source/ignore_unused_var4.bal

```diff
@@ -0,0 +1,9 @@
+type Person record {|
+    string name;
+    int id;
+|};
+
+function foo(Person person) returns int {
+    Person {id, name} = person;
```

Comment:
The text edit produces a syntax error.
![erro](https://user-images.githubusercontent.com/35211477/139922304-a4fe46ee-8684-4cf9-bb5c-031768b06987.png)

Reply:
Thanks for pointing that out. My bad. Fixed the logic.